Formula/charmcraft.rb

```diff
@@ -0,0 +1,210 @@
+class Charmcraft < Formula
+  include Language::Python::Virtualenv
+
+  desc "Shiny new formula"
```

Comment:
Can we get a slightly more informative description?

Reply:
Sorry about this, not sure how I missed it

Reply:
You seem to have missed it again

Reply:
done now, sorry about that